setup.py

```diff
@@ -22,12 +22,13 @@
     long_description=long_description,
     test_suite='tests',
     tests_require=tests_require,
-    platforms='any',
     install_requires=install_requires,
     classifiers=[
         'Intended Audience :: Developers',
         'License :: OSI Approved :: MIT License',
-        'Operating System :: OS Independent',
+        'Operating System :: MacOS',
+        'Operating System :: MacOS :: MacOS X',
+        'Operating System :: POSIX :: Linux',
```

Comment:
Why not also include `Operating System :: POSIX`? I am not aware of any Linux-specific code that we're using. Are you?

Reply:
Nope, I just wasn't sure what the correct tags were so I guessed.